client/components/PubEdgeListing/mixins.scss

```diff
@@ -0,0 +1,7 @@
+@mixin hover-top-border() {
```

Comment:
Nit — the name `hover` here makes it seem like this has something to do with hover states. I assume it's chosen because it "hovers" above its box-layout position? In that case, I might consider `float`...oh, wait...